frontends/web/src/routes/account/summary/coinbalance.tsx

```diff
@@ -50,6 +56,22 @@ export function CoinBalance ({
   const accountsPerCoin = getAccountsPerCoin();
   const coins = Object.keys(accountsPerCoin) as accountApi.CoinCode[];
 
+  const coinsWithLightning: TCoinMap = {} as TCoinMap;
+  coins.forEach(coinCode => {
+    if (accountsPerCoin[coinCode]?.length >= 1) {
+      const account = accountsPerCoin[coinCode][0];
+      coinsWithLightning[account.coinCode] = account.coinName;
+    }
+  });
+
+  const { lightningConfig } = useLightning();
+  if (lightningConfig.accounts.length > 0) {
+    const bitcoinCode = 'btc';
+    const bitcoinName = 'Bitcoin';
+    if (!coinsWithLightning[bitcoinCode]) {
+      coinsWithLightning[bitcoinCode] = bitcoinName;
+    }
+  }
```

Comment:
coinsWithLightning is confusing me, but maybe it's only the name?

I find it hard to read. It creates a map here, and then later turns the map into Object.keys and Object.entries. We already receave another map in this component coinsBalances and have a helped function to create another map accountsPerCoin.. feel like this all could be simplified, but I am not sure.

Couldn't you just make a `const totals = [];` array instead of the map and push coins/total in there somehow? 

Also we already have `coinsBalances` map 🤔

Reply:
Yeah, the name may be confusing.

eg. when you have only a Litecoin account enabled the `accountsPerCoin` will only contain this account, so I created `coinsWithLightning`to add Bitcoin there (when Lightning is active and if `accountsPerCoin` doesn't include it yet)

`coinsBalances` can't be used as they contain just balances, not active accounts. If the amounts are 0 on the accounts, we want to show them. It also doesn't include `coinName`.

Reply:
yeah, you are right, it can be simplified to use only `coinsBalances`. I have to include the coin name there.

Reply:
We miss the coin name in `coinsBalances`...

Reply:
I agree, this is kind of confusing to me, too. Also, I think that if some info is missing, it should come from the backend in a simple and straight-forward way, as a general approach.

Reply:
this OR simplifies the logic and should come first: https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2714

Reply:
ok there is some nice stuff in my PR, but the coinName and order of coins/tokens shouldn't be job of the frontend. So I will close the PR, but maybe take some TS improvements and make a separate PR.